src/views/Swap.vue

```diff
@@ -455,7 +451,7 @@ export default {
       return this.networkMarketData[this.asset]
     },
     ethRequired () {
-      return this.networkWalletBalances.ETH === 0
+      return this.assetChain === 'ETH' && this.networkWalletBalances.ETH === 0
```

Comment:
`assetChain` would be only the from asset right?
If to asset is also on eth network , you need ETH too.
so i think it should be `[this.assetChain, this.toAssetChain].includes('ETH') && this.networkWalletBalances.ETH === 0`